Chessington.GameEngine/Square.cs

```diff
@@ -11,6 +11,32 @@ public Square(int row, int col)
             Col = col;
         }
 
+        public bool IsSquareInsideBorders()
```

Comment:
I think by default all squares are inside the borders and pieces can move out of borders, so I believe that this method should belong to the piece class and not square.